appserver/extras/docker-images/src/main/resources/payaraserver-node/entrypoint.sh

```diff
@@ -4,15 +4,105 @@ set -e
 DOCKER_CONTAINER_ID="$(cat /proc/self/cgroup | grep :/docker/  | sed s/\\//\\n/g | tail -1)"
 echo "Docker Container ID is: ${DOCKER_CONTAINER_ID}"
 
+if [ -z ${DOCKER_CONTAINER_IP} ]; then
+    echo "No Docker container IP override given, setting to first result from 'hostname -I'"
+    DOCKER_CONTAINER_IP="$(hostname -I | cut -f1 -d ' ')"
+    echo "Hostname is ${DOCKER_CONTAINER_IP}"
+fi
+
+echo "Docker Container IP is: ${DOCKER_CONTAINER_IP}"
+
 ### Functions ###
+function checkAndCreateNewNodeIfRequired {
+    if [ -z ${PAYARA_NODE_NAME} ]; then
```

Comment:
Missing quotation marks

Reply:
Are they necessary? I haven't got them anywhere else and it works fine from my testing.

Reply:
https://stackoverflow.com/questions/51112152/how-to-accept-a-quoted-sentence-as-argument-in-shell-script - I didn't try, but I expect that if I would set the property with spaces inside, instead of an error reponse the script would crash.